src/platforms/javascript/common/configuration/micro-frontend-support/module-metadata-integration.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: ModuleMetadata Integration
+sidebar_order: 20
+description: "Read injected metadata and attach it to relevant error stack frames."
+---
+
+Requires SDK version `7.59.0` or higher.
+
+The `ModuleMetadata` integration looks up module metadata that has been <PlatformLink
```

Comment:
Instead of starting the page with what the ModuleMetadata integration does, could we start it with why a user would want to add it?

Something like:

"The `ModuleMetadata` integration enables you to tag or route events based on which JavaScript bundle the error came from..."